### DISCUSSION TITLE
Overview of the evidence for New Physics and of anomalies

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#249] [Astrophysics and Physics]
This post is a listing of empirical evidence for New Physics mechanisms and of various measurements anomalies.
While connecting the dots of the puzzle is hard, empirical anomalies have 1) reached high sigma significance and 2) are the substrate for which to base on cosmological ideas and 3) are absolutely key to the ambitious goal of designing an homeostatic (matter-radiation recycling) universe, which most likely require new neutrino physics.

Xenon1T
https://www.semanticscholar.org/paper/Constraints-on-electron-scattering-interpretation-Chigusa-Endo/383a5378c2ab13a1ac17e9b4b3cdbda5630b0206
> Finally, to conclude with another interesting sociological case, one can remind when XENON1T published a presumed excess above the threshold as reported in Fig 15 [120], and later non confirmed by XENONnT with a lower overall background [71]. 

The Solar neutrino problem and related Gallium anomaly 
https://arxiv.org/abs/2306.03299

Upcoming resolution of the neutrino mass ordering
https://www.semanticscholar.org/paper/Earliest-Resolution-to-the-Neutrino-Mass-Ordering-Cabrera-Han/33166ed419fe19f8ecca402238f5d06bdd5a4b2d

https://www.semanticscholar.org/paper/DARWIN-XLZD%3A-A-future-xenon-observatory-for-dark-Baudis/55c18451657db798a2b1b69c1f76a5e111d06b17/figure/0

https://arxiv.org/abs/2310.20472

> The kinematics implies that the most promising mass range for the new hypothetical particles to be produced by collisions at NICA is subGeV. The new physics, needed to explain neutrino oscillations, dark matter, baryon asymmetry of the Universe, inflation, etc, can indeed be at such low energy scale, provided by only feeble interaction with known ingredients of the Standard Model of particle physics (SM)

https://arxiv.org/html/2401.12893v2

"Figure 1:The regions to be probed at NICA"
![image](https://github.com/user-attachments/assets/26a9ca57-e954-47c7-9b66-853d2eefdafe)

![image](https://github.com/user-attachments/assets/b266490d-06d5-479c-8d3b-782fc1c2ceb6)

https://web.archive.org/web/20201126154003/https://pdfs.semanticscholar.org/22c0/158e9d4f5e1f85484aec7150e44b996dbf1a.pdf

https://en.m.wikipedia.org/wiki/Super_Charm-Tau_factory
http://www.scholarpedia.org/article/The_Panda_experiment

https://ufn.ru/en/articles/2024/1/e/
See figure 5

https://arxiv.org/abs/2101.11877
https://www.prophy.science/article/174857612/

https://arxiv.org/abs/2209.08041

**Comment by LifeIsStrange** [comment]
New physics from Gamma Ray bursts?

In 2022 was observed the highest energy Gamma ray burst (~70 time for powerful than previous record), the first Pev scale burst (1.4 Pev))
I don't know what is the frequency of PEV grbs, I don't think we could observe them before LHASOO but at least regarding other parameters like it's duration, the 2022 burst is thought to be a one of a kind event every 10000 years (probably wrong but still rare) 

A crucial question in cosmology and for the CMB is: How transparent is the universe?
We know that the ISM is matter rich because of the dispersion measure of fast radio bursts.

The higher the energy/light frequency, the more likely light is to interact with matter (e.g radio waves pass through walls while rayons X are carcinogenic).

Therefore there should be an absolute limit to the highest energy photons that reach us, based on their distance because they should interact and lose their energy/be scattered during their travel in the IGM/ISM.

https://web.archive.org/web/20230816124909/https://www.nationalgeographic.com/premium/article/brightest-blast-ever-seen-in-space-continues-to-surprise-scientists

> According to current theories, these highest-energy photons shouldn’t be able to pass through the universe for two billion years, as seen from GRB 221009A. However, one way to detect these signals is if the very high-energy gamma rays [convert into a hypothetical particle of dark matter called an axion](https://web.archive.org/web/20230816124909/https://arxiv.org/abs/2210.05659), which then travels unimpeded through the universe and converts back once it reaches our galaxy’s magnetic field.

Dark matter or not I wonder how strong is the evidence for this being driven by a new particle, what would be other explanations?

https://arxiv.org/abs/2109.07794
The amaterasu particle is one order of magnitude more energetic
https://arxiv.org/abs/2405.03528
> This single particle appears to have emerged, inexplicably, from the [Local Void](https://en.m.wikipedia.org/wiki/Local_Void), an empty area of space bordering the [Milky Way](https://en.m.wikipedia.org/wiki/Milky_Way) galaxy.[[2]](https://en.m.wikipedia.org/wiki/Amaterasu_particle#cite_note-2)

**Comment by LifeIsStrange** [comment]
This paper seems to casually drop that the Higgs boson standard model particle recently discovered, seems to no longer fit the experimental data we took as evidence for it and instead the anomaly is better explained by new physics?
https://arxiv.org/abs/2402.02822

**Reply by mikehelland** [reply]
![image](https://github.com/user-attachments/assets/2601a485-cb59-4c20-ad34-ddd72c65cc94)

From [2014](https://www.explainxkcd.com/wiki/index.php/1437:_Higgs_Boson).

**Reply by LifeIsStrange** [reply]
LOL I find it incredible the amount of relatable comics XKCD has done
Now on a pedantic note:
I'm not sure this implies the standard higgs boson is wrong, more that it is incomplete (CP violation?, interaction with an invisible particle, three distinct Higgs bosons, etc)
Moreover about the money begging side meme,
There is a popular blog skeptical about the usefulness of an LHC successor and this idea was already well popularized by Sabine hosseinfield.
Can't find it back but here is another
https://bigthink.com/hard-science/large-hadron-collider-economics/
I don't remember all their arguments but IMO I tended to agree before but now that I have read a bit the literature on particle physics, it appears clear to me that most of the arguments are fallacious or misleading.
Arguments:
The LHC successor cost arround 20 billions and won't be built until a long time (mostly because of funding delays), all this funding absorb the rest of the funding that could be allocated to "small scale"/non particle Colliders physics experiments that would still advance particle physics.
This argument actually has some merits, there are many non Collider experiments that have extreme usefulness, be it the various neutrinos detectors, the dark matter detectors, the high energy cosmic rays detectors, the experiments that constraint the magnetic dipole of particles (literally more impactful yet complementary than Colliders), the tests of gravity and many very specific experiments (Gallium anomaly, etc).
While funding could be improved, I'd argue that most of the interesting projects are already funded so this is a moot point.

The other argument being that, the standard model is too successful, there are no anomalies/evidence for New Physics nor are there predicted new particles to detect at the specific range of next generation colliders.
This is utterly wrong, there is a dozen of real major anomalies in particle physics that are robust evidence for sterile neutrinos or new forces or new particles.
Moreover there are actually in addition to dark matter axions and sterile neutrinos, other theorized particles to constrain, even in the standard model orthodoxy, be it Higgs boson triplets, neutrinos tridents, the neutrino mass ordering, wether fractional charges exists, lorentz invariance and an often forgotten but credible one: Glueballs.
What IS true however is that going higher in energy is not guaranteed to help that much.
The Higgs boson is a good counter example as it only manifest at high energy (hundreds of GEVs)
But what people considerably under appreciate is that the revolution we awaits in particle physics does NOT come from a larger LHC successor.
Because besides energy, the most major Bottleneck is to increase precision, to improve the signal to noise ratio of accelerators, and to increase their expressivity (the kinds of reactions they can observe)
For that there are a few independent and complementary methods:
Increasing the luminosity independently of the energy. (Belle II recently broke the record and is making major discoveries) 
Using an electron instead of a hadron Collider, which is much better despite reaching lower energies. NICA and belle II should revolutionize that.
Using a linear accelerator (FAIR?, ILC) 
Developing specific colliders types such as the next Tau factory, antiparticle factory, etc
Also there is the promise of gluon colliders but I am skeptical on that one.

We live in a critical period for particle physics where anomalies and new physics are being confirmed in real time and it is those low energy projects that will revolutionize it.
I have high hopes for the future accelerators that will simultaneously reach LHC like energy and better accuracy but they will only come in 10-15 years, in the meantime we should make major discoveries that let's hope, will accelerate funding.

**Comment by LifeIsStrange** [comment]
https://en.m.wikipedia.org/wiki/CONUS-Experiment
I just learned about Neutrino coherent elastic scattering, which increases it's cross section by one order of magnitude and therefore neutrino detectability.

I wonder how this will compete with DUNE
https://arxiv.org/abs/2407.11912
https://arxiv.org/abs/2405.05554
